sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/cli/OwaspZapScanExecutor.java

```diff
@@ -39,15 +38,29 @@ public void execute(OwaspZapScanContext scanContext) throws ZapWrapperRuntimeExc
 
     }
 
+    private void assertTargetReachable(OwaspZapScanContext scanContext) {
+        boolean isReachable = false;
+        for (String url : scanContext.getOwaspZapURLsIncludeList()) {
+            URI uriToCheck = URI.create(url);
```

Comment:
This can go wrong, then you get an exception. I would prefer doing the conversation before the connection check. I would actually expect `getOwaspZapURLsIncludeList` to return URLs and not strings.